<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StickyNote</class>
 <widget class="QMainWindow" name="StickyNote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StickyNote</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>45</y>
      <width>395</width>
      <height>360</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="addBtn">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>3</y>
      <width>36</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="topBtn">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>3</y>
      <width>36</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="editBtn">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>3</y>
      <width>36</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="minBtn">
    <property name="geometry">
     <rect>
      <x>302</x>
      <y>3</y>
      <width>36</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="settingBtn">
    <property name="geometry">
     <rect>
      <x>343</x>
      <y>3</y>
      <width>36</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteBtn">
    <property name="geometry">
     <rect>
      <x>384</x>
      <y>3</y>
      <width>36</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>3</y>
      <width>165</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>3</y>
      <width>165</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
